config/jobs/kubernetes-sigs/kind/kind-postsubmits.yaml

```diff
@@ -10,3 +10,10 @@ postsubmits:
       - image: gcr.io/k8s-testimages/kubekins-e2e:v20180924-8e8468033-experimental
```

Comment:
Which base image is this using? Would be great to check if the underlying library would respect the ndots option we set :)

Reply:
https://github.com/kubernetes/test-infra/tree/master/images/kubekins-e2e ->
https://github.com/kubernetes/test-infra/tree/master/images/bootstrap ->
`FROM debian:jessie`

Reply:
Great, debian uses glibc, which I think does repsect the configuration in `/etc/resolv.conf`.